rust/cli/src/commands/run.rs

```diff
@@ -92,6 +92,14 @@ fn display_run_message(config: &FrontendServerConfig) {
             .underline()
             .blue()
     );
+    println!(
+        "\n☁️ {} Chroma Cloud powers serverless vector and full-text \
```

Comment:
I like it - it's just a little long. 

@jeffchuber do you want to refine the copy? Otherwise, I can take a pass. 

Catching you up - at standup we discussed how npm + pip install commands wouldn't work for advertising Cloud, but Itai suggested advertising it in the `chroma run` logs.